starter/source/output/th/write_thnms1.F90

```diff
@@ -401,6 +405,8 @@ subroutine write_thnms1(nvarn1        ,nvarn1a       ,nvarn2           ,nvarnpin
           call write_thnms1_titles(io,nvars8,vars8_title,vars8,239030)
           call write_thnms1_titles(io,nvars9,vars9_title,vars9,239030+nvars8)
           call write_thnms1_titles(io,nvarsnloc,varsnloc_title,varsnloc,239030+nvars8+nvars9)
+          call write_thnms1_titles(io,nvars10,vars10_title,vars10,239030+nvars8+nvars9+nvarsnloc)
+          call write_thnms1_titles(io,nvars10,vars10_title,vars10,239030+nvars8+nvars9+nvarsnloc)
```

Comment:
@Schwoertzig 

I think this line is dupplicated

Reply:
Great catch, thank you so much for reviewing the code! It is updated